src/dnf-plugins/product-id/product-id.c

```diff
@@ -315,6 +361,21 @@ void getEnabled(const GPtrArray *repos, GPtrArray *enabledRepos) {
     }
 }
 
+/**
+ * Find the list of repos that are actually disabled
+ * @param repos all available repos
+ * @param disabledRepos the list of disabled repos
```

Comment:
The name `disabledRepos` doesn't match up with the parameter's actual name.  The name `enabledRepos` in the method seems like a misnomer too.